packages/storage/src/chrome/local.ts

```diff
@@ -35,10 +42,36 @@ export const localDefaults: LocalStorageState = {
   knownSites: [{ origin: MINIFRONT_URL, choice: UserChoice.Approved, date: Date.now() }],
 };
 
+interface Migration {
+  wallets: {
+    [LocalStorageVersion.V1]: (old: LocalStorageState['wallets']) => LocalStorageState['wallets'];
+  };
+}
+
+const migrations: Migration = {
+  wallets: {
+    [LocalStorageVersion.V1]: old =>
+      old.map(({ fullViewingKey, id, label, custody }) => {
+        const fvk = new FullViewingKey({
+          inner: bech32ToFullViewingKey(fullViewingKey),
+        });
+        const walletId = new WalletId({
+          inner: bech32ToWalletId(id),
+        });
+        return {
+          fullViewingKey: fvk.toJsonString(),
+          id: walletId.toJsonString(),
+          label,
+          custody,
+        };
+      }),
```

Comment:
Can you move this to a new v1-migration.ts file?

```typescript
const migrations: Migration = {
  wallets: {
    [LocalStorageVersion.V1]: v1WalletMigration
  },
};
```

Also, can you write tests for this specific migration as well?